console/servlets/build.gradle

```diff
@@ -0,0 +1,23 @@
+distsDirName = 'webapps'
+
+plugins.apply 'war'
+
+dependencies {
+  providedCompile 'javax.servlet:javax.servlet-api:3.1.0'
+  providedCompile project(':utils:streamscope')
+  providedCompile ext_classpath
+}
+
+war {
```

Comment:
Yes, all the tests passed for me. The change in 5a5014f is needed to find the console.war file built in the gradle build. Make sure you have the complete branch checked out.